src/tests/nativeaot/SmokeTests/HelloWasm/HelloWasm.cs

```diff
@@ -373,6 +373,8 @@ private static unsafe int Main(string[] args)
 
         TestBoolCompare();
 
+        TestGetDependenciesDueToAccess();
```

Comment:
I'll leave it up to you but if you're enabling the Dataflow tests, they'll cover this scenario too and the failure mode of the test is probably easier to troubleshoot.

Reply:
Thanks, removed as suggested.